### ISSUE TITLE
Attempt to create credentials for publicKey fail with error

### ISSUE THREAD

**Issue by Gozala** [beakerbrowser/beaker#1137]
Operating system:
macOS

This is:

- [x] a bug
- [ ] general feedback

As per our discussion over fritter and comment from @RangerMauve I tried to create keypair as described on [corresponding MDN documentation](https://developer.mozilla.org/en-US/docs/Web/API/PublicKeyCredentialCreationOptions). But seem to get  **DOMException: Could not establish connection to the credential manager.**

<img width="667" alt="screen shot 2018-09-04 at 1 44 00 pm" src="https://user-images.githubusercontent.com/21236/45058790-95aed200-b04e-11e8-89aa-bb3a50ad8f6b.png">

Don't know if that's elector issue or beaker needs to wire something up, but it seems worth reporting.

**Comment by RangerMauve** [comment]
Yeah, I don't think Beaker (or maybe electron as you say) implements the needed APIs to support this.

Sadly, I don't know much about electron (in the context of browser engines like beaker) so I don't know if it provides hooks for this stuff.

**Comment by RangerMauve** [comment]
Seems this is the electron bug tracking it: https://github.com/electron/electron/issues/7150

**Comment by pfrazee** [comment]
Thanks for reporting, sorry for the issue ☹️. We'll either get this fixed or provide a temporary solution.
